app/controllers/hiring_staff/vacancies/application_details_controller.rb

```diff
@@ -31,7 +31,7 @@ def edit
 
   # rubocop:disable Metrics/AbcSize
   def update
-    vacancy = school.vacancies.published.find(vacancy_id)
+    vacancy = school.vacancies.find(vacancy_id)
```

Comment:
Is this change required? As only published vacancies need to be updated (I think `create` updates any drafts)